sdks/wasm/Makefile

```diff
@@ -75,18 +75,24 @@ managed/nunitlite.dll: $(TOP)/mcs/class/lib/wasm/nunitlite.dll | managed/
 	cd $(TOP)/mcs/tools/nunit-lite && make PROFILE=wasm
 	cp $< $@
 
+<<<<<<< HEAD
```

Comment:
remove conflict marker

Reply:
uggg, I had stashed some random changes that unfortunately has the fix for it :/